opba-facade-protocol-api-shared/pom.xml

```diff
@@ -22,5 +22,9 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Why this one is needed?

Reply:
was problem with property name for serialization/deserialization. But changing property name made this dependency unnecessary